src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -101,6 +106,10 @@ export class AssetsTable extends React.Component {
     return this.trashcanRefs[focusAsset.id];
   }
 
+  getImageThumbnail(thumbnail) {
+    return thumbnail ? (<img src={`${this.props.courseDetails.base_url}${thumbnail}`} alt="Description not available" />) : (<img src={undefined} alt="Preview not available" />);
```

Comment:
I'm kind of surprised the linter isn't complaining about this line. It's definitely longer than 100 characters...